examples/hello-pt/app/config/config_fed_server.json

```diff
@@ -40,6 +36,13 @@
     }
   ],
   "workflows": [
+      {
+        "id": "pre_train",
+        "name": "GlobalWeightInitializer",
```

Comment:
Since we have "ScatterAndGather" as a workflow name,

Should we change this name "GlobalWeightInitializer" to
"InitializeGlobalWeights" ?

Reply:
Sure. Makes sense.